test/e2e/scenario/vulnerability-overflow/00-assert.yaml

```
@@ -10,3 +10,15 @@ spec:
 status:
   vulnerabilitySummary:
     severityCount: {}
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+collectors:
+  - type: events
+    namespace: image-scanner-jobs
+  - type: pod
+    namespace: image-scanner
+    pod: trivy-0
+  - type: pod
+    namespace: image-scanner
+    selector: control-plane=image-scanner
```

Could we add the `kubectl describe node` as well? 
```suggestion
    selector: control-plane=image-scanner
  - type: command
    command: kubectl describe node
```